libs/scripts/scripts.js

```diff
@@ -44,7 +44,34 @@ export async function loadLCP(blocks) {
   }
 }
 
-function loadDelayed() {}
+/**
+ * loads a script by adding a script tag to the head.
+ * @param {string} url URL of the js file
+ * @param {Function} callback callback on load
+ * @param {string} type type attribute of script tag
+ * @returns {Element} script element
+ */
+export function loadScript(url, callback, type) {
```

Comment:
This function is already in uitls.js

Reply:
I went and removed the loadScript duplicate function definition after seeing it in the utils.js file, thanks for pointing it out.